pywbem/cim_operations.py

```diff
@@ -1709,6 +1721,269 @@ def OpenEnumerateInstancePaths(self, ClassName, namespace=None,
                 self.operation_recorder.record_staged()
             return result_tuple
 
+    # TODO overriding the internal choice of pull vs enum. Should we be able
+    #      force one or the other for a connection?
+    # TODO is this a good name for the generator function
+    def EnumerateInstancesGen(self, ClassName, namespace=None,
```

Comment:
Some material on discussing naming conventions for generator functions:
- https://bytes.com/topic/python/answers/506531-generator-naming-convention
- https://stackoverflow.com/questions/9039265/name-of-a-function-returning-a-generator

I think given that we have different flavors of methods, we cannot just use no prefix or no suffix. It seems to me that using an `iter` prefix would be most Pythonese. Maybe in mixed case to be consistent with our naming style, i.e. `Iter`. Basically, use the name of the open operation and replace `Open` with `Iter`.